framework_tvb/tvb/interfaces/web/static/js/bursts.js

```diff
@@ -435,6 +437,28 @@ function setPseRangeParameters(){
     });
 }
 
+function setArrowKeyEventListener(){
+    document.addEventListener('keyup', (e) => {
+        // if focus is not on any of the current fragment's elements, set it on the next or launch button
+        const focused_element = document.activeElement;
+        if(focused_element.tagName === 'BODY'){
+            const simulator_form = document.getElementById('div-simulator-parameters');
+            setInitialFocusOnButton(simulator_form);
+        }else {
+            if (e.code === 'ArrowRight'){
```

Comment:
I think that if we leave the default (access next by TAB) behaviour, then we only need to make sure that attribute `tabindex ` is properly set in the HTML and that the initial focus is set correctly.
This needs to be checked. Something on this direction already happens in the 1.5.* versions https://www.thevirtualbrain.org/burst/

Reply:
I updated my code. It all works fine.

Only one observation: I have checked the 1.5 version and it's true that tabindex="1" is set on each element but pressing tab takes you through all elements, including the simulations from burst history, the main menu from the lower part of the page and so on, so it's not just the elements from the form. And here I set tabindex only on the buttons cause I don't know how to put it on the inputs, select fields etc. cause they are dinamically created. But even without doing that, it still functions the same way as in the 1.5 version.